integration/e2e/step-definitions/browse.steps.ts

```diff
@@ -36,7 +36,7 @@ defineSupportCode(function ({ Given, When, Then }) {
 
   Then(/^the content of the page (.*) is displayed$/, async (pageNumber: string) => {
```

Comment:
The title say that the **content** on a given page should be displayed, but there is no check that the content actually is displayed. It only checks that the page number is correct.